README.md

```diff
@@ -1,146 +1,143 @@
 
-   ```console
-   echo "nextflow run main.nf  -profile singularity --input treeval.yaml" | bsub -Is -tty -e error -o out -n 10 -q normal -M10000 -R'select[mem>10000] rusage[mem=10000] span[hosts=1]'
-   ```
+```bash
+nextflow run main.nf -profile singularity --input treeval.yaml -entry {FULL|RAPID} --outdir {OUTDIR}
+```
 
 ## Documentation
 
-The nf-core/treeval pipeline comes with documentation about the pipeline [usage](https://nf-co.re/treeval/usage), [parameters](https://nf-co.re/treeval/parameters) and [output](https://nf-co.re/treeval/output).
+The sanger-tol/treeval pipeline comes with documentation about the pipeline [usage](https://nf-co.re/treeval/usage), [parameters](https://nf-co.re/treeval/parameters) and [output](https://nf-co.re/treeval/output).
+
+> **Warning:**
+> Please provide pipeline parameters via the CLI or Nextflow `-params-file` option. Custom config files including those
+> provided by the `-c` Nextflow option can be used to provide any configuration _**except for parameters**_;
+> see [docs](https://nf-co.re/usage/configuration#custom-configuration-files).
 
 ## Credits
 
-nf-core/treeval was originally written by Damon-Lee Pointon (@DLBPointon), Yumi Sims (@yumisims) and William Eagles (@weaglesBio).
+sanger-tol/treeval has been written by Damon-Lee Pointon (@DLBPointon), Yumi Sims (@yumisims) and William Eagles (@weaglesBio).
 
 We thank the following people for their extensive assistance in the development of this pipeline:
 
-@muffato
-@gq1
-@ksenia-krasheninnikova
-@priyanka-surana
+<ul>
+  <li>@muffato</li>
+  <li>@gq1</li>
+  <li>@ksenia-krasheninnikova</li>
+  <li>@priyanka-surana</li>
+  <li>@mcshane</li>
+</ul>
 
 ## Contributions and Support
 
 If you would like to contribute to this pipeline, please see the [contributing guidelines](.github/CONTRIBUTING.md).
 
-For further information or help, don't hesitate to get in touch on the [Slack `#treeval` channel](https://nfcore.slack.com/channels/treeval) (you can join with [this invite](https://nf-co.re/join/slack)).
-
 ## Citations
 
-<!-- If you use  nf-core/treeval for your analysis, please cite it using the following doi: [10.5281/zenodo.XXXXXX](https://doi.org/10.5281/zenodo.XXXXXX) -->
+<!--TODO: Citation-->
 
-### Tools
-
-BedTools
-
-Bionano CMAP
-
-BUSCO
-
-Minimap2
+If you use sanger-tol/treeval for your analysis, please cite it using the following doi: [10.5281/zenodo.XXXXXX](https://doi.org/10.5281/zenodo.XXXXXX).
 
-Miniprot
-
-Mummer
-
-Python3
-
-Samtools
-
-TABIX
+### Tools
 
-UCSC
+<ul>
+  <li>BedTools</li>
+  <li>BUSCO </li>
+  <li>bwa-mem2</li>
+  <li>Cooler</li>
+  <li>Find Telomere<//li>
+  <li>Juicer</li>
+  <li>Minimap2</li>
+  <li>Miniprot</li>
+  <li>Mummer</li>
+  <li>Python3</li>
+  <ul>
+    <li>Pyfasta</li>
+    <li>Pandas</li>
+    <li>Pybedtools</li>
+  </ul>
+  <li>Perl</li>
+  <li>Pretext</li>
+  <li>Samtools</li>
+  <li>Staden io_lib</li>
+  <li>Tabix</li>
+  <li>UCSC_Tools - Kent</li>
+  <li>WindowMasker</li>
+
+</ul>
```

Comment:
I don't think this is needed, since you have them in CITATIONS already.

Reply:
Removed